app/src/main/res/layout-sw600dp/help_activity.xml

```diff
@@ -10,9 +10,23 @@
     android:layout_height="match_parent"
     android:orientation="vertical">
 
-    <include
-      android:id="@+id/help_activity_toolbar"
-      layout="@layout/toolbar" />
+    <com.google.android.material.appbar.AppBarLayout
```

Comment:
This code doesn't match what's currently in toolbar.xml--why the differrence?

Reply:
All the toolbars in all the affected(banned) XML files, now have code that's the same as in toolbar.xml. PTAL

Reply:
I still see two differences at least here: the extra layout_gravity end, and the reversion in background color being used. Also, this is wrapped in AppBarLayout but it wasn't before. Could you please correct these here & elsewhere in the PR?